frontend/packages/local-storage-operator-plugin/src/constants/index.ts

```diff
@@ -2,6 +2,13 @@ export const diskModeDropdownItems = Object.freeze({
   BLOCK: 'Block',
   FILESYSTEM: 'Filesystem',
 });
+
+export const deviceTypeDropdownItems = Object.freeze({
+  DISK: 'Disk',
+  PART: 'Part',
+  BOTH: 'Both',
```

Comment:
I'll not have `both` option rather we have had multiple selections dropdown or first value as non selected which means both will be allowed. 

Can you check if deviceType is not provided what happens?

Reply:
Yes, there should be multi select dropdown instead of having "Both" as option

Reply:
yap, I agree, multi-selection works